### ISSUE TITLE
Fix accessibility test

### ISSUE THREAD

**Issue by poteirard** [zopaUK/react-components#1]
## The problem

After the migration from the Github Enterprise to open source we had to redo the yarn.lock.

This caused some accessibility tests failures.

Affected files:
`src/components/molecules/TextField/TextField.test.tsx`
`src/components/molecules/CheckboxField/CheckboxField.test.tsx`
`src/components/molecules/ZopaFooter/ZopaFooter.test.tsx`
`src/components/molecules/RadioField/RadioField.test.tsx`
`src/components/organisms/Accordion/AccordionHeader/AccordionHeader.test.tsx`
`src/components/molecules/DropdownFiltered/DropdownFiltered.test.tsx`

At the moment I commented/skip them to make the CI/CD work.

## The task
Investigate why are failing and fix them

**Comment by lluia** [comment]
I'm going to work on this 👌

**Comment by lluia** [comment]
It looks like it has something to do with the [latest patch on `jest-axe`](https://github.com/nickcolley/jest-axe/issues/56#issuecomment-467801560).
